icd/generated/mock_icd.cpp

```diff
@@ -3568,6 +3698,12 @@ static VKAPI_ATTR void VKAPI_CALL CmdDrawIndexedIndirectCountAMD(
 
 
 
+#ifdef VK_ENABLE_BETA_EXTENSIONS
+#endif /* VK_ENABLE_BETA_EXTENSIONS */
+
+#ifdef VK_ENABLE_BETA_EXTENSIONS
+#endif /* VK_ENABLE_BETA_EXTENSIONS */
+
```

Comment:
Looks like the empty `ifdef`s are throughout the code.

Reply:
The video extension is actually multiple extensions, many of which have no entry points. So the code generator will open/close the platform block on `beginFeature()`/`endFeature()` callbacks, but never receive any callbacks or emit any code in between them. This is not unique to `VK_ENABLE_BETA_EXTENSIONS`, you can also find this pattern for some of the `VK_USE_PLATFORM_WIN32` and `VK_USE_PLATFORM_GGP` ifdefs

Reply:
Ah, okay.